src/platform/file_manager_cache.h

```diff
@@ -5,11 +5,14 @@
 #define USE_FILE_CACHE
 
 #include "core/file.h"
+#include <stdbool.h>
```

Comment:
Same here `stdbool` doesn't have to be used.

Reply:
```suggestion
```